tests/beman/exemplar/CMakeLists.txt

```diff
@@ -10,3 +10,31 @@ target_link_libraries(
 )
 
 gtest_add_tests(beman.exemplar.tests.identity "" AUTO)
+
+if(BEMAN_EXEMPLAR_INSTALL_CONFIG_FILE_PACKAGE)
+    block()
+        if(UNIX)
+            add_test(
+                NAME beman.exemplar.test.config-file-package
+                COMMAND
+                    "${PROJECT_SOURCE_DIR}/cmake/test/test.sh"
+                    "${CMAKE_COMMAND}"
+                    "${PROJECT_SOURCE_DIR}/cmake/test" # source dir
+                    "${CMAKE_CURRENT_BINARY_DIR}/cmake/test" # build dir
+                    "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}" # prefix path
+                    "$<CONFIG>" "-DCMAKE_CXX_STANDARD=17"
+            )
+        else()
+            add_test(
+                NAME beman.exemplar.test.config-file-package
+                COMMAND
+                    "${PROJECT_SOURCE_DIR}/cmake/test/test.bat"
+                    "${CMAKE_COMMAND}"
+                    "${PROJECT_SOURCE_DIR}/cmake/test" # source dir
+                    "${CMAKE_CURRENT_BINARY_DIR}/cmake/test" # build dir
+                    "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}" # prefix path
+                    "$<CONFIG>"
+            )
```

Comment:
I don't like to call bat as sh scripts, because this is not portable.

And you see, this if else block should to the same, but it does't!

Too the `CMAKE_CXX_STANDARD` is a user defined value!

For `CXX_MODULES`, the build and test environment mast be absolute the same.